lib/cli/run-option-metadata.js

```diff
@@ -49,7 +49,7 @@ const TYPES = (exports.types = {
     'sort',
     'watch'
   ],
-  number: ['retries', 'jobs'],
+  number: ['retries', 'jobs', 'jsonStringifyWhitespace'],
```

Comment:
Minor feedback: This seems to be an unrelated addition

Reply:
This relates to #5017 and involves introducing an optional --jsw flag that allows users to specify the amount of whitespace in JSON reporter output. But, I'll exclude this from the current pr to focus on addressing one feature addition.